app/code/Magento/Checkout/Test/Mftf/Test/StorefrontDeleteConfigurableProductFromWebsiteMiniShoppingCartTest.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontDeleteConfigurableProductFromWebsiteMiniShoppingCartTest">
```

Comment:
That's really awesome that you have created an MFTF test. It will be very helpful for checking the expected behavior.
I have a minor recommendation on how to make it more consistent. Basically the current version checks that after all the minicart is empty. It might be tricky since the shopping cart may be empty because of many consequences (i.e. the session was killed). I would rather add a couple of products to the shopping cart, then delete one of them from the store, then check if the cart is NOT empty and the deleted item is NOT present. 

I hope it will help. Thank you.